src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -62,6 +63,16 @@ module _ (S : Setoid c ℓ) where
   ∉-resp-≋ : ∀ {x} → (x ∉_) Respects _≋_
   ∉-resp-≋ xs≋ys v∉xs v∈ys = v∉xs (∈-resp-≋ (≋-sym xs≋ys) v∈ys)
 
+  -- index is injective in its first argument.
+
+  index-injective
+    : ∀ {x₁ x₂ xs} (x₁∈xs : x₁ ∈ xs) (x₂∈xs : x₂ ∈ xs) →
+      Any.index x₁∈xs ≡ Any.index x₂∈xs → x₁ ≈ x₂
```

Comment:
This does not respect the style guide, cf.:
https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#indentation